client/components/common/tce-core/QuestionContainer/index.vue

```diff
@@ -147,6 +147,16 @@ export default {
       this.$set(element.data, 'feedback', { ...element.data.feedback, ...data });
     }
   },
+  watch: {
+    element: {
+      handler() {
+        this.isEditing = !this.element.id;
```

Comment:
This flips newly added assessment in the standard container into 'saved' mode (since once BE responds it 'autosaves').